src/resources/language/_language-pt.yml

```diff
@@ -51,3 +51,22 @@ crossref-eq-prefix: "Equação"
 crossref-lof-title: "Lista de Figuras"
 crossref-lot-title: "Lista de Tabelas"
 crossref-lol-title: "Lista de Listagens"
+
+listing-page-order-by: "Ordenar por"
+listing-page-order-by-default: "Pré-selecionado"
+listing-page-order-by-date-asc: "O mais novo"
+listing-page-order-by-date-desc: "Mais velho"
+listing-page-order-by-number-desc: "Descendente"
+listing-page-order-by-number-asc: "Ascendente"
+listing-page-field-date: "Encontro"
```

Comment:
```
listing-page-field-date: "Data"
```

("Encontro" translates to "will you go on a `date`" :)

Reply:
That wasn't quite what I had in mind!